# If position error (goal and current motor pos difference) 
# is bigger than error_threshold an object is grasped
position_error_threshold: 0.06

